webhook/server/server.go

```diff
@@ -174,7 +174,9 @@ func (m *podInjector) Handle(ctx context.Context, req admission.Request) admissi
 
 	dkVol := oa.Spec.CodeModules.Volume
 	if dkVol == (corev1.VolumeSource{}) {
-		dkVol.EmptyDir = &corev1.EmptyDirVolumeSource{}
+		dkVol.CSI = &corev1.CSIVolumeSource{
+			Driver: "csi.oneagent.dynatrace.com",
```

Comment:
It's a bit of a nitpick, but a constant should be used for the driver name

Reply:
There is already one: `dtcsi.DriverName` in `controllers/csi/config.go`